docs/data-management/edf-to-bids.md

```diff
@@ -537,6 +419,149 @@ if not calibration.empty:
         warn("Calibration data found but unsuccessfully parsed for results")
 ```
 
+### Parsing the `recording` dataframe
```

Comment:
Why the reordering? One problem with reordering is that the diff is not anymore informative (everything is green and we can't see specific changes)

Reply:
The only changes apart from the reordering is the addition of the global variable definition in the code snippet (l. 501-527)